src/main/java/com/zufar/onlinestore/cart/converter/ShoppingSessionItemDtoConverter.java

```diff
@@ -5,10 +5,11 @@
 import com.zufar.onlinestore.product.converter.ProductInfoDtoConverter;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
 import org.mapstruct.Named;
 
 
-@Mapper(componentModel = "spring", uses = ProductInfoDtoConverter.class)
+@Mapper(componentModel = MappingConstants.ComponentModel.SPRING, uses = ProductInfoDtoConverter.class)
```

Comment:
maybe better use "spring" in the componentModel instead of MappingConstants.ComponentModel.SPRING ?

Reply:
no